Backpack/src/main/java/net/skyscanner/backpack/text/BpkTextField.kt

```diff
@@ -102,9 +108,6 @@ open class BpkTextField @JvmOverloads constructor(
     setTextColor(
       colorStateList(
         color = textColor,
-        pressedColor = textColor,
```

Comment:
I've added a default param in the `colorStateList` for these so these aren't needed anymore